submissions/2020-11-11-OpenFF-Optimization-Set-1-charged/README.md

```diff
@@ -0,0 +1,52 @@
+### Description
+
+This is an optimization dataset from Gen 1 fitting [https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2019-05-16-Roche-Optimization_Set]() with the protomers and tautomers enumerated.
+Conformers were generated using a RMS cutoff of 3 Ångstroms.
+
+### General Information
+ - Date: 2020.11.11
+ - Class: OpenFF Optimization dataset
+ - Purpose: Improve coverage of FF with addition of protonation states and tautomers
+ - Collection: OptimizationDataset
+ - Name: OpenFF Optimization Set 1 charged
+ - Number of unique molecules: 468
```

Comment:
Please can you update this with the number calculated using `dataset.n_molecules` as we have created some new tautomers.

Reply:
Sure Josh, I had a doubt about whether the number of input molecules should be the no. of unique molecules so I placed this number as a placeholder, will update that